### PR TITLE
added/scrollbar

### PR DESCRIPTION
**Pull request by itsmeabhishek03**
*Have you read the [Contributing Guidelines](https://github.com/jessesquires/.github/blob/master/CONTRIBUTING.md)?* 
Yes

Issue #47

## Describe your changes
I made a small change in style.css file in client directory. Now it looks like this - 
image

image

### CODE REVIEW

**Comment by anand-harsh** [comment]
Hi @itsmeabhishek03 , Kindly change the color of scrollbar to yellow in light mode and white in dark mode

**Review by anand-harsh — CHANGES_REQUESTED** [review]
Do the changes told for the colors

**Comment by itsmeabhishek03** [comment]
> Hi @itsmeabhishek03 , Kindly change the color of scrollbar to yellow in light mode and white in dark mode

ok

**Comment by anand-harsh** [comment]
@itsmeabhishek03 , RESOLVE THE ISSUE ASAP, its already up the deadline, if you are late, points wont get counted

**Comment by anand-harsh** [comment]
Hi @itsmeabhishek03 any updates?

**Comment by anand-harsh** [comment]
Hi @itsmeabhishek03 , your deadline is over, either you finish it by tonight, or this will be merged and you won't get the points

File: client/src/style.css [hunk 1]
```diff
@@ -1,3 +1,20 @@
...
+}
+
+::-webkit-scrollbar-track{
+    border-radius: 5px;
+    box-shadow: inset 0 0 10px rgba(0,0,0,0.25);
+}
+
+::-webkit-scrollbar-thumb{
+    border-radius: 5px;
+    background: linear-gradient(to top, #f57811 , #1d08ba);
```
Comment (anand-harsh) [inline client/src/style.css:16]: add yellow background here for scrollbar

**Review by anand-harsh — CHANGES_REQUESTED** [review]
Do the changes

**Pull request closed without merging**